src/main/kotlin/no/nav/medlemskap/services/tpsws/PersonMapper.kt

```diff
@@ -53,13 +52,15 @@ fun mapPersonhistorikkResultat(personhistorikkResponse: HentPersonhistorikkRespo
                     adresselinje = it.strukturertAdresse.tilleggsadresse ?: "Ukjent adresse",
                     landkode = it.strukturertAdresse.landkode.value,
                     fom = it.postleveringsPeriode.fom.asDate(),
-                    tom = it.postleveringsPeriode.tom.asDate()
+                    tom = it.postleveringsPeriode.tom.asDate(),
+                    adresseType = hentMidertidigPostadresserBasertPaaLandekode(it.strukturertAdresse.landkode.value)
```

Comment:
Her er det vel egentlig gitt ut fra klassen hvilken adressetype det er? Iom at du er inne i `is MidlertidigPostadresseNorge` ? Tilsvarende nedenfor, med `is MidlertidigPostadresseUtland`. Så kanskje unødvendig å kalle `hentMidertidigPostadresserBasertPaaLandekode` her?

Reply:
Ja, var litt i tvil på denne, men det er nok nok det